src/richText.js

```diff
@@ -749,8 +749,8 @@ define([
                     (isFormRef ? '<div class="popover-content"><p></p></div>' : '') +
                     '</div></div>',
                 delay: {
-                    show: 0,
-                    hide: 200,
+                    show: 350,  // be less annoying
```

Comment:
Slightly related: Does this mean that the analytics event is fired after 350 ms instead of instantaneously on hover?